Formula/qwtpolar.rb

```diff
@@ -0,0 +1,94 @@
+class Qwtpolar < Formula
+  desc "Library for displaying values on a polar coordinate system"
+  homepage "http://qwtpolar.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/qwtpolar/qwtpolar/1.1.1/qwtpolar-1.1.1.tar.bz2"
+  sha256 "6168baa9dbc8d527ae1ebf2631313291a1d545da268a05f4caa52ceadbe8b295"
+
+  bottle do
+    cellar :any
+    sha256 "e51aec713366e7406d63b0eea55f41385a24d67ff7d298f9fd479ae14dea2e3c" => :el_capitan
+    sha256 "8bd14ade82bd28887ec4bfef8098cd893806b380b2176eae6885ec5da4168a54" => :yosemite
+    sha256 "cb47115b5ca12d61ccc63a2cd681323825f6a73001a02f62a1f58c8a920ae82d" => :mavericks
+  end
+
+  option "without-plugin", "Skip building the Qt Designer plugin"
+
+  depends_on "qt5"
+  depends_on "qwt"
+
+  # Update designer plugin linking back to qwtpolar framework/lib after install
+  # See: https://sourceforge.net/p/qwtpolar/patches/2/
```

Comment:
Looks like from that that there was a 1.2.0 release. Wonder why we never saw a tarball? I'm fine with reinstating given that but would be good to submit some of this stuff upstream and see if we can nag them into a release.